client/src/components/evals/eval-runner.tsx

```
@@ -766,10 +936,52 @@ export function EvalRunner({
           </div>
         );
       case "review":
+        const totalTestCases = validTestTemplates.length * selectedModels.length;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)